dtmf_dolphin_data.c

```diff
@@ -80,6 +80,17 @@ DTMFDolphinSceneData DTMFDolphinSceneDataRedboxUS = {
     }
 };
 
+DTMFDolphinSceneData DTMFDolphinSceneDataRedboxCA = {
+    .name = "Redbox (CA)",
+    .block = DTMF_DOLPHIN_TONE_BLOCK_REDBOX_CA,
+    .tone_count = 3,
+    .tones = {
+      {"Nickel",  2200.0, {0, 0, 5}, 1, 66, 0},
```

Comment:
@LambdaCalculus37 thanks for putting in these values! To work with the approach I've taken so far, these rows will need a 0 value after the tone in the sequence; even single tone data is stored as dual tone with the second tone set to 0.

Reply:
Okay, I'll make the appropriate changes so you can review later.